addons/point_of_sale/static/src/app/store/product_configurator_popup/product_configurator_popup.js

```diff
@@ -25,10 +25,20 @@ export class BaseProductAttribute extends Component {
 
         return {
             value,
-            valueId: selected_value.id,
+            valueIds: selected_value.id,
             extra: selected_value.price_extra,
         };
     }
+
+    updateMultipleAttribute(event) {
+        const value = parseInt(event.target.id);
+
+        if (this.state.checkbox_value.has(value)) {
+            this.state.checkbox_value.delete(value);
+        } else {
+            this.state.checkbox_value.add(value);
+        }
+    }
```

Comment:
It's complicated to manage several checkboxes with a single t-model, so using the function is more appropriate.

Reply:
what can be simpler than [this](https://odoo.github.io/owl/playground/#eyJqcyI6ImltcG9ydCB7IENvbXBvbmVudCwgdXNlU3RhdGUsIG1vdW50IH0gZnJvbSBcIkBvZG9vL293bFwiO1xuY2xhc3MgRm9ybSBleHRlbmRzIENvbXBvbmVudCB7XG4gIHN0YXRpYyB0ZW1wbGF0ZSA9IFwiRm9ybVwiO1xuICBzZXR1cCgpIHsgICAgXG4gICAgdGhpcy5udW1zID0gWzEsMiwzLDQsNSw2LDcsOF07XG4gICAgdGhpcy5zdGF0ZSA9IHVzZVN0YXRlKHtib29sczogdGhpcy5udW1zLm1hcCh4PT5mYWxzZSl9KTtcbiAgfVxufVxubW91bnQoRm9ybSwgZG9jdW1lbnQuYm9keSwgeyB0ZW1wbGF0ZXM6IFRFTVBMQVRFUywgZGV2OiB0cnVlIH0pO1xuIiwiY3NzIjoiIiwieG1sIjoiPHRlbXBsYXRlcz5cbiAgPGRpdiB0LW5hbWU9XCJGb3JtXCI+XG4gICAgPGlucHV0IHQtZm9yZWFjaD1cInRoaXMubnVtc1wiIHQtYXM9XCJpXCIgdC1rZXk9XCJpXCIgdHlwZT1cImNoZWNrYm94XCIgdC1tb2RlbD1cInN0YXRlLmJvb2xzW2ktMV1cIi8+XG4gICAgPGRpdiB0LWZvcmVhY2g9XCJ0aGlzLm51bXNcIiB0LWFzPVwiaVwiIHQta2V5PVwiaVwiIHQtZXNjPVwiaSArICc6ICcgK3N0YXRlLmJvb2xzW2ktMV1cIj48L2Rpdj5cbiAgPC9kaXY+XG48L3RlbXBsYXRlcz4ifQ==)? 
![image](https://github.com/odoo/odoo/assets/118446179/95bb7eb5-91cd-44a6-90c7-8b6da91be776)

Reply:
you can also use an object instead of an array in this example

Reply:
I think it's the same thing, and in the code base, both methods are used.

Reply:
it's not the same thing. The whole idea of t-model is that you could just use you own handler function, but that the t-model makes everything so much simpler. It's not like it doesn't work without t-model. It's just that it would allow you to remove 100 lines of code very easily and also to have to deal with less bugs in the future. It's true that there are many places where we use handler functions instead of t-model, but that's not necessarily a good thing. There used to be many places where we were using jquery for example